pkg/rancher-desktop/typings/electron-ipc.d.ts

```diff
@@ -118,6 +119,10 @@ export interface IpcMainInvokeEvents {
   /** Fetch data from the backend, or arbitrary host ignoring CORS. */
   'extensions/vm/http-fetch': (config: import('@docker/extension-api-client-types').v1.RequestConfig) => any;
   // #endregion
+
+  // #region Versions
+  'versions/macOs': () => semver.SemVer;
```

Comment:
Hate to quibble, but this should be called `macOS` to match the official case

Reply:
@ericpromislow and I discussed out of band and concluded that this can remain as is. I quibbled the same  while writing this and opted for consistency with `getMacOsVersion`.